python.sh

```diff
@@ -1,30 +1,24 @@
 package: Python
 version: "%(tag_basename)s"
-tag: alice/v2.7.10
-source: https://github.com/alisw/cpython.git
+tag: v3.6.8
+source: https://github.com/python/cpython
 requires:
  - AliEn-Runtime:(?!.*ppc64)
  - FreeType
  - libpng
  - sqlite
+ - "GCC-Toolchain:(?!osx)"
 build_requires:
  - curl
 env:
-  SSL_CERT_FILE: "$(export PATH=$PYTHON_ROOT/bin:$PATH; export LD_LIBRARY_PATH=$PYTHON_ROOT/lib:$LD_LIBRARY_PATH; python -c \"import certifi; print certifi.where()\")"
+  SSL_CERT_FILE: "$(export PATH=$PYTHON_ROOT/bin:$PATH; export LD_LIBRARY_PATH=$PYTHON_ROOT/lib:$LD_LIBRARY_PATH; python -c \"import certifi; print(certifi.where())\")"
   PYTHONHOME: "$PYTHON_ROOT"
-prefer_system: (?!slc5)
```

Comment:
This effectively removes the `prefer_system`, I guess you meant `prefer_system: .*`